docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,109 @@
+---
+id: manage-connector-templates
+title: Manage Connector templates
+description: Manage your Connector templates in Web Modeler.
+---
+
+export const UploadIcon = () => <span style={{verticalAlign: "text-top"}}><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16"><path d="M0 0h24v24H0z" fill="none"></path><path d="M9 16h6v-6h4l-7-7-7 7h4zm-4 2h14v2H5z" fill="currentColor"></path></svg></span>;
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+You can create and manage [Connector templates](/components/integration-framework/connectors/custom-built-connectors/connector-templates.md) just as any other asset in a Web Modeler project.
+
+## Create Connector templates
+
+Take the following steps to create a new Connector template::
+
+1. Navigate to the project of your choice and click **New**.
+
+2. Click **Connector Template**.
+
+   ![Creating a new template](img/connector-templates/create-connector-template-1.png)
+
+3. You will be taken to the **Connector template editor** screen. In this screen, you can define the Connector template by writing the template JSON. The template editor supports you in writing the template by providing autocompletion, error highlighting, and a live preview.
+
+   The anatomy of the editor interface is as follows:
```

Comment:
Minor tweak:
Update text from "The anatomy of the editor interface is as follows:"
to "The components of the editor interface are as follows:"